third_party/move/move-compiler-v2/tests/recursive-instantiation-checker/v1-tests/complex_1.exp

```diff
@@ -0,0 +1,32 @@
+
+Diagnostics:
+error: cyclic type instantiation
+   ┌─ tests/recursive-instantiation-checker/v1-tests/complex_1.move:13:9
+   │
+13 │     fun c<T1, T2>() {
+   │         ^
+   │
+   = `c<T1, T2>` calls `d<#1>` at tests/recursive-instantiation-checker/v1-tests/complex_1.move:13
+   = `d<#1>` calls `b<u64, T2>` at tests/recursive-instantiation-checker/v1-tests/complex_1.move:15
```

Comment:
It would be nice if we could change `#1` to `T1` here.  Can you figure out why it is being displayed that way?

Reply:
fixed

Reply:
Awesome!